src/components/TextInput/styles.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components';
+
+export const InpuText = styled.input`
+  width: 300px;
+  height: 35px;
+  outline: none;
+  margin: 0;
+  margin-left: 5px;
+  padding: 6px;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  background: #ffffff;
+  border: 1px solid #808080;
+  border-radius: 8px;
+`;
+export const LabelText = styled.label`
+  color: #808080;
+  font-family: Open Sans;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+`;
```

Comment:
Please set the styles for the different states of the input, you can check them out at Figma file, in the components page:

<img width="826" alt="Captura de Pantalla 2020-08-26 a la(s) 8 01 20 a  m" src="https://user-images.githubusercontent.com/17033709/91306579-56fe0f00-e772-11ea-97ca-960d9ff49a86.png">

Reply:
the focus will ok, for the select, si not done on this component